src/distributed-transaction/introduction.md

```diff
@@ -0,0 +1,13 @@
+# Introduction
+
+TiKV is a distributed transactional key-value database, transaction is a foundational function for TiKV. This chapter we will talk about some concepts about distributed transaction and some also provide some details about distributed transaction implementation in TiKV.
```

Comment:
"concepts about distributed transactions" (since we're talking about general implementations, plural), but the later "distributed transaction" is fine. (since we're talking about a specific, singular implementation)